mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -832,7 +850,7 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 		ARG 2 yaw
 	METHOD m_yhvbjlmg setPosition (Lnet/minecraft/unmapped/C_vgpupfxx;)V
 		ARG 1 pos
-	METHOD m_ykrmhbpu getEntityWorld ()Lnet/minecraft/unmapped/C_cdctfzbn;
+	METHOD m_ykrmhbpu world ()Lnet/minecraft/unmapped/C_cdctfzbn;
```

Comment:
This method conflicts with `m_dggmledz`

Reply:
Urgh, why is Enigma

`m_dggmledz` is supposed to be named `serverWorld` but it forces me to name it the same for no reason?

Reply:
That's odd